cmd/agent/dist/conf.d/winkmem.d/conf.yaml.example

```diff
@@ -0,0 +1,35 @@
+init_config:
+  ## The Windows kernel memory check lists the top users of kernel memory by pool tag.
+  ## for each of the four metrics, configure the top N results to be reported.  The
+  ## default for all is 10.
+
+  ## @param topPagedBytes: - integer - optional - default: 10
+  ## report the top N pool tag consumers by size of paged memory
+  # topPagedBytes: 8
+
+  ## @param topNonPagedBytes: - integer - optional - default: 10
+  ## report the top N pool tag consumers by size of non-paged memory
+  # topNonPagedBytes: 5
+
+  ## @topPagedAllocsOutstanding: - integer - optional - default: 10
+  ## report the pool tag with the top number of allocations outstanding 
+  ## (allocations minus frees).  This can be different from the topPagedBytes
+  ## if there are many small allocations (vs. a few large)
+  # topPagedAllocsOutstanding: 5
+
+  ## @topNonPagedAllocsOutstanding: - integer - optional - default: 10
+  ## report the pool tag with the top number of non-paged allocations outstanding
+  ## (allocations minus frees).  This can be different from the topNonPagedBytes
+  ## if there are many small allocations (vs. a few large)
+  # topNonPagedAllocsOutstanding: 5
+
+  ## @alloctags: - list - optional - default: None
+  ## list of specific pool tags that should be reported regardless of whether
+  ## that tag is a top consumer
+  # alloctags:
```

Comment:
This lists pool tags, perhaps the parameter would be better named `pool_tags`, or `report_extra_pool_tags` even? Also, do you think it would make sense to mention/link to pooltag.txt in the description?